### DISCUSSION TITLE
Can someone help me with a json for 4 ch sonoff sprinkler

### DISCUSSION THREAD

**Discussion by mariusvasss** [RavenSystem/esp-homekit-devices#1508] [MEPLHAA Scripts]
I want them to work in a row with a 5 min cycle one after other, it is possible?

**Comment by RavenSystem** [comment]
Yes, it is possible. You must use service notification actions (`"m":[]`) to turn on next valve service when a valve finishes. If you have a working JSON with 4 valves, I could add them.

**Reply by mariusvasss** [reply]
I've made it. Thank you a lot!!
